Update customer-key-set-up.md (#7946)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/customer-key-set-up.md

### Create a premium Azure Key Vault in each subscription

The steps to create a key vault are documented in [Getting Started with Azure Key Vault](/azure/key-vault/general/overview), which guides you through installing and launching Azure PowerShell, connecting to your Azure subscription, creating a resource group, and creating a key vault in that resource group.

> SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 

Reviewer: ```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```
```suggestion
SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 
```

> SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 

Reviewer: ```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```
```suggestion
SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 
```

> SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 

Reviewer: ```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```

>  Azure Key Vault provides two types of containers, 
> 
> 
> 1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   
> 
> 2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 
> 
>  SPO supports both Standard/Premium vaults as well as Managed HSM.

Reviewer: ```suggestion
 Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```
```suggestion
 Azure Key Vault provides two types of containers, 

1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SPO supports both Standard/Premium vaults as well as Managed HSM.
```

>  Azure Key Vault provides two types of containers, 
> 
> 
> 1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   
> 
> 2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 
> 
>  SPO supports both Standard/Premium vaults as well as Managed HSM.

Reviewer: ```suggestion
 Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```
```suggestion
 Azure Key Vault provides two types of containers, 

1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SPO supports both Standard/Premium vaults as well as Managed HSM.
```

>  Azure Key Vault provides two types of containers, 
> 
> 
> 1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   
> 
> 2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 
> 
>  SPO supports both Standard/Premium vaults as well as Managed HSM.

Reviewer: ```suggestion
  Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2.

SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```

>  Azure Key Vault provides two types of containers, 
> 
> 
> 1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   
> 
> 2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 
> 
>  SPO supports both Standard/Premium vaults as well as Managed HSM.

Reviewer: ```suggestion
  Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2.

SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.```
